react/Carousel.js

```diff
@@ -123,20 +123,20 @@ export default class Carousel extends Component {
       typeOfRoute === 'external' ? {
         page: {
           type: 'string',
-          title: 'Banner link',
+          title: 'Banner link (should start with https)',
```

Comment:
I think that is better to remove this `(should start with https)` and put a default value `https://`